helevents/templates/rest_framework/event_list.html

```diff
@@ -134,6 +134,19 @@ <h3 id="event-language">Event language</h3>
 <pre><code>event/?language=ru
 </code></pre>
 <p><a href="?language=ru" title="json">See the result</a></p>
+<h3 id="audience-age">Event audience age boundaries.</h3>
```

Comment:
Here, the problem (and the reason why this has not been documented thus far) is that we actually have zero events which have the audience age data available in the production API at the moment. Therefore, this filter doesn't do anything in production.

Therefore, only when we have event data that contains audience age for some sensible subset of events, we can add documentation for this filter in production, *with* the caveat that most events *don't* contain this data at all. All the other documented filters actually work for all events.

Reply:
Okay, perhaps wording "to find events that are designed for specific age audiences" is enough 🤔 That explains to the API user why the filter doesn't do anything?

Reply:
Not exactly 0... https://api.hel.fi/linkedevents/v1/event/?audience_min_age=16 kulke and harrastushaku support audience_min/max_age, this is needed for harrastuspassi. I will update the docs.